script/e2e-test.sh

```diff
@@ -67,12 +67,25 @@ k8s_ensure_image kubeapps kubeapps-ci-internal-dashboard $DEV_TAG
 k8s_ensure_image kubeapps kubeapps-ci-internal-tiller-proxy $DEV_TAG
 
 # Wait for Kubeapps Pods
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci-internal-apprepository-controller
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci-internal-chartsvc
-k8s_wait_for_pod_ready kubeapps app=kubeapps-ci-internal-tiller-proxy
-k8s_wait_for_pod_ready kubeapps app=mongodb
-k8s_wait_for_pod_completed kubeapps apprepositories.kubeapps.com/repo-name=stable
+k8s_wait_for_deployment kubeapps kubeapps-ci
```

Comment:
One of the reason why in https://github.com/kubeapps/kubeapps/pull/833 I decided to not to pass the name of the deployment was to make the script less error prone and be sure that we always test all the deployments in place. So if we add a new service, its rollout will be tested regardless if the user forgot to update the ci scripts.

That said, @andresmgot convinced me that doing it explicitly has the benefit of being sure that the expected services are there. So I am ok keeping this pattern.

What I think we should do though is to refactor this a little bit and use a for loop.